src/Serilog.Sinks.PeriodicBatching/Sinks/PeriodicBatching/BoundedConcurrentQueue.cs

```diff
@@ -6,22 +6,20 @@ namespace Serilog.Sinks.PeriodicBatching
 {
     class BoundedConcurrentQueue<T> 
     {
-        const int NON_BOUNDED = -1;
+        public const int NON_BOUNDED = -1;
```

Comment:
Perhaps we should use the normal `NonBounded` casing here, to keep the codebase consistent?